crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -170,7 +170,7 @@ function CrowdsourceHinter(runtime, element, data){
         $.each(result, function(index, value) {
             if(value != "Reported"){
                 showStudentSubmissionHistory(value);
-                student_answer = value;
+                student_answer = encodeURI(value);
```

Comment:
an attempt to fix the spacing situation that was unsuccessful. should be changed back to previous.